hadoop-ozone/dist/src/main/smoketest/s3/awss3.robot

```diff
@@ -47,7 +47,13 @@ File upload and directory list
                         Should not contain        ${result}         dir1
                         Should contain            ${result}         file
 
+List buckets with empty access id
+                        Execute                    aws configure set aws_access_key_id ''
+    ${result} =         Execute AWSS3APICli and checkrc         list-buckets    255
```

Comment:
S3 tests are grouped, mostly based on operation type.  `list-buckets` belongs to `bucketlist.robot`.

(`awss3.robot` is for `aws s3` command, while other tests are for `aws s3api`.)

Also, adding it as last test case avoids the need to run `Setup s3 tests` explicitly in the next test case.

Reply:
Sure, I'll move it to bucketlist.robot.
I didn't put it at last incase others adding new test would loss the accesskey.